client/packages/inventory/src/Stocktake/ListView/CreateStocktakeModal.tsx

```diff
@@ -201,127 +153,96 @@ export const CreateStocktakeModal = ({
-                  labelProps={{ sx: { flex: `${LABEL_FLEX}` } }}
-                  Input={
-                    <VVMStatusSearchInput
-                      disabled={
-                        !!createBlankStocktake || includeAllMasterListItems
-                      }
-                      onChange={vvmStatus =>
-                        setState(prev => ({
-                          ...prev,
-                          vvmStatus: vvmStatus ?? null,
-                        }))
-                      }
-                      width={380}
-                      selected={vvmStatus}
-                    />
-                  }
-                />
-              )}
+                  <FormControlLabel
+                    value={StocktakeType.BLANK}
+                    control={<Radio sx={{ paddingY: '7px' }} />}
+                    label={t('stocktake.create-blank')}
+                    slotProps={{ typography: { fontWeight: 'bold' } }}
+                  />
+                </RadioGroup>
 
-              <InputWithLabelRow
-                labelProps={{ sx: { flex: `${LABEL_FLEX}` } }}
-                Input={estimatedLineCount}
-                label={t('label.stocktake-estimated-lines')}
-              />
+                <Box
+                  sx={{
+                    backgroundColor: 'background.group.light',
+                    padding: 2,
+                    borderRadius: '10px',
+                  }}
+                >
+                  {type === StocktakeType.FULL && (
+                    <>
+                      <Typography variant="body2">
+                        {t('stocktake.description-full')}
+                      </Typography>
+                      <FullStocktakeOnHandSelector
+                        includeAllItems={includeAllItems}
+                        setState={setState}
+                      />
+                    </>
+                  )}
+                  {type === StocktakeType.FILTERED && (
+                    <>
+                      <Typography variant="body2">
+                        {t('stocktake.description-filters')}
+                      </Typography>
+                      <StocktakeFilters state={state} setState={setState} />
+                    </>
+                  )}
+                  {type === StocktakeType.BLANK && (
+                    <Typography variant="body2">
+                      {t('stocktake.description-blank')}
+                    </Typography>
+                  )}
+                </Box>
+              </Box>
+
+              {/* Estimated lines */}
+              {createBlankStocktake ? (
+                <Alert severity="success" sx={{ marginRight: 0 }}>
+                  {t('message.create-blank-stocktake')}
+                </Alert>
+              ) : (
+                <Alert severity="info" sx={{ marginRight: 0 }}>
+                  {t('message.lines-estimated', {
+                    count: estimateLineCount(),
```

Comment:
Just wondering why this is an estimate, as opposed to the definite value?

Reply:
It's probably pretty accurate now... used to just the bigger number between master list lines and stock lines, which was often wrong, if there was a mix of both 👀 

I'm kind of alright with keeping the estimate phrasing, gives us a bit of wiggle room, the stocktake filters are quite complex, what do you reckon?

Reply:
Yeah, seems fine, just wondering :)